File: sentimental-mario-less/mario.py
# TODO
while True:
    try:
        n = int(input("Height: "))
        if n >= 1 and n <= 8:
            break
    except ValueError:
        continue
# output used

# loops
for i in range(1, n + 1):
    for j in range(n - i):
        print(" ", end="")
    for k in range(i):
        print("#", end="")
    print("\n", end="")

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // input
    int n;
    do
    {
        n = get_int("Height: ");
    }

    while (n < 1 || n > 8);

    //output used

    //loops
    for (int i = 1; i <= n; i++)
    {
        for (int k = 1; k <= (n - i); k++)
        {
            printf(" ");
        }
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}